dist/source/repo-creator/repository-chooser.html

```diff
@@ -28,17 +28,29 @@
 					</ul>
 				</div>
 			</div>
+            <div class="app-search-error">
+                    <div class="message">
+                        <div class="text bg-danger">
+                            <i class="fa fa-warning"></i> Oops something went wrong! Please try after some time.
+                        </div>
+                    </div>
+                </div>
```

Comment:
Can you switch these to tabs instead of spaces for indentation?  The rest of the file is using tab indents except for this new block.  What editor do you use for HTML?  There is a `.editorconfig` in the project if your editor supports it that will tell your editor to use tab instead of space.